packages/fx-core/src/plugins/resource/aad/aadAppClient.ts

```diff
@@ -47,6 +46,9 @@ export class AadAppClient {
       config.clientId = provisionAadResponse.appId;
       config.objectId = provisionAadResponse.id;
     } catch (error) {
+      if (error?.response?.status >= 400 && error?.response?.status < 500) {
```

Comment:
It is better to use constant to replace 400 and 500.

Reply:
Fixed. Thank you for pointing that out.